rosidl_typesupport_opendds_cpp/RawData.idl

```diff
@@ -0,0 +1,7 @@
+const long KEY_HASH_LENGTH_16 = 16;
+
+struct RawData {
+	@key octet       key_hash[KEY_HASH_LENGTH_16];
+	sequence<octet>  key_data;
+	sequence<octet>  raw_data;
```

Comment:
OpenDDS doesn't support anonymous types at the moment.

Reply:
@kuznetsovmoci How do you want to handle this situation? It's what Connext has in their code but not supported in OpenDDS.

Reply:
Well for sequences of primitive types like `octet` you need to use some predefined types in TAO. Also you will need to add `@topic` to the struct or change the options on `opendds_idl` to make all types topic types by default.

```
#include <tao/OctetSeq.pidl>

const long KEY_HASH_LENGTH_16 = 16;

@topic
struct RawData {
	@key octet key_hash[KEY_HASH_LENGTH_16];
        OctetSeq key_data;
	OctetSeq raw_data;
};
```

Reply:
I updated based on what @mitza-oci  and I did for opendds_static_serialized_data.idl.